src/simmate/apps/bader/workflows/bader.py

```diff
@@ -24,3 +25,16 @@ class PopulationAnalysis__Bader__Bader(S3Workflow):
     density that is not associated with any atomic orbital. For these cases,
     you will see files like "CHGCAR_w_empty_atoms" used in the command.
     """
+
+    @staticmethod
+    def setup(directory, **kwargs):
+        """
+        The henkelman bader algorithm uses the total charge density as a reference
+        file. VASP returns the core electrons and valence electrons in seperate
+        files which must be summed together to create this reference file. This
+        setup method performs this action and writes the necessary file
+        """
+        aeccar0 = Grid.from_file(directory / "AECCAR0")
+        aeccar2 = Grid.from_file(directory / "AECCAR2")
```

Comment:
not for this PR, but I bet you eventually want a feature to allow setting these and/or let the user provide their own sum (kinda how we did in our early versions of badelf)